features/admins/admin_skins.feature

```diff
@@ -0,0 +1,114 @@
+@skins
+Feature: Admin manage skins
+
+  Scenario: Users should not be able to see the admin skins page
+  Given I am logged in as "skinner"
+  When I go to admin's skins page
+  Then I should see "I'm sorry, only an admin can look at that area"
+
+  Scenario: Admin can cache and uncache a public skin
```

Comment:
With a pull that ends up this big it would be worth doing the refactor as a separate pull I think so we don't have to think about all the things at once.